fitlins/workflows/base.py

```diff
@@ -269,6 +271,7 @@ def _deindex(tsv):
             name='ds_{}_contrast_plots'.format(level))
 
         if ix == 0:
+            model.inputs.drop_missing = drop_missing
```

Comment:
This should be for `design_matrix`. Maybe just set it in the `DesignMatrix()` call?